modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+A key feature to enable as part of single node installation is workload partitioning.
+This limits the cores allowed to run platform services maximizing the CPU core for
+application payloads. You must configure workload partitioning at cluster installation time.
+
+[NOTE]
+====
+Worlkload partitioning is required to be applied during installation.
+====
+
+.Procedure
+
+* No configuration changes are needed. Use the provided settings:
```

Comment:
Not true. The base64 encoded content below contains the CPU set that the management workloads are constrained to. These must be adjusted to match the set specified in the performanceprofile and must be accurate for the number of cores on the cluster. See telcodocs-299 for more details.

Reply:
@imiller0 Discuss wording.

Reply:
Make the wording change. Add xref in assembly.